docker/docker-compose.yml

```diff
@@ -3,7 +3,16 @@ version: '2'
 services:
   conductor-server:
     environment:
-      - CONFIG_PROP=config.properties
+      - db=dynomite
+      - workflow_dynomite_cluster_hosts=dyno1:8102:us-east-1c
+      - workflow_dynomite_cluster_name=dyno1
+      - workflow_namespace_prefix=conductor
+      - workflow_namespace_queue_prefix=conductor_queues
+      - queues_dynomite_threads=10
+      - queues_dynomite_nonQuorum_port=22122
+      - workflow_elasticsearch_url=es:9300
+      - workflow_elasticsearch_index_name=conductor
```

Comment:
I'm not sure I understand.  I get the UPPERCASE part.  Are you saying that, rather than checking the environment variable first, we simply overwrite the values in the actual configuration file at startup?  I think some of the Elastic.co products behave that way.  For example, if you run `kibana:5.2` and set the `ELASTICSEARCH_URL` environment variable, I'm pretty sure that it actually writes that value into the kibana.yml as `elasticsearch.url`.